app/i18n/en-us/gen.php

```diff
@@ -0,0 +1,215 @@
+	),
+	'auth' => array(
+		'accept_tos' => 'I accept the <a href="%s">Terms of Service</a>.',
+		'email' => 'Email address',
+		'keep_logged_in' => 'Keep me logged in <small>(%s days)</small>',
+		'login' => 'Login',
+		'logout' => 'Logout',
+		'password' => array(
+			'_' => 'Password',
+			'format' => '<small>At least 7 characters</small>',
+		),
+		'registration' => array(
+			'_' => 'New account',
+			'ask' => 'Create an account?',
+			'title' => 'Account creation',
+		),
+		'reset' => 'Authentication reset',
+		'username' => array(
+			'_' => 'Username',
+			'admin' => 'Administrator username',
+			'format' => '<small>Maximum 16 alphanumeric characters</small>',
+		),
+	),
+	'date' => array(
+		'Apr' => '\\A\\p\\r\\i\\l',
+		'Aug' => '\\A\\u\\g\\u\\s\\t',
+		'Dec' => '\\D\\e\\c\\e\\m\\b\\e\\r',
+		'Feb' => '\\F\\e\\b\\r\\u\\a\\r\\y',
+		'Jan' => '\\J\\a\\n\\u\\a\\r\\y',
+		'Jul' => '\\J\\u\\l\\y',
+		'Jun' => '\\J\\u\\n\\e',
+		'Mar' => '\\M\\a\\r\\c\\h',
+		'May' => '\\M\\a\\y',
+		'Nov' => '\\N\\o\\v\\e\\m\\b\\e\\r',
+		'Oct' => '\\O\\c\\t\\o\\b\\e\\r',
+		'Sep' => '\\S\\e\\p\\t\\e\\m\\b\\e\\r',
+		'apr' => 'Apr.',
+		'april' => 'April',
+		'aug' => 'Aug.',
+		'august' => 'August',
+		'before_yesterday' => 'Before yesterday',
+		'dec' => 'Dec.',
+		'december' => 'December',
+		'feb' => 'Feb.',
+		'february' => 'February',
+		'format_date' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y',
+		'format_date_hour' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y \\a\\t H\\:i',
+		'fri' => 'Fri',
+		'jan' => 'Jan.',
+		'january' => 'January',
+		'jul' => 'July',
+		'july' => 'July',
+		'jun' => 'June',
+		'june' => 'June',
+		'last_2_year' => 'Last two years',
+		'last_3_month' => 'Last three months',
+		'last_3_year' => 'Last three years',
+		'last_5_year' => 'Last five years',
+		'last_6_month' => 'Last six months',
+		'last_month' => 'Last month',
+		'last_week' => 'Last week',
+		'last_year' => 'Last year',
+		'mar' => 'Mar.',
+		'march' => 'March',
+		'may' => 'May',
+		'may_' => 'May',
+		'mon' => 'Mon',
+		'month' => 'months',
+		'nov' => 'Nov.',
+		'november' => 'November',
+		'oct' => 'Oct.',
+		'october' => 'October',
+		'sat' => 'Sat',
+		'sep' => 'Sept.',
+		'september' => 'September',
+		'sun' => 'Sun',
+		'thu' => 'Thu',
+		'today' => 'Today',
+		'tue' => 'Tue',
+		'wed' => 'Wed',
+		'yesterday' => 'Yesterday',
+	),
+	'dir' => 'ltr',
+	'freshrss' => array(
+		'_' => 'FreshRSS',
+		'about' => 'About FreshRSS',
+	),
+	'js' => array(
+		'category_empty' => 'Empty category',
+		'confirm_action' => 'Are you sure you want to perform this action? It cannot be canceled!',
+		'confirm_action_feed_cat' => 'Are you sure you want to perform this action? You will lose related favorites and user queries. It cannot be canceled!',
+		'feedback' => array(
+			'body_new_articles' => 'There are %%d new articles to read on FreshRSS.',
+			'request_failed' => 'A request has failed, it may have been caused by internet connection problems.',
```

Comment:
The funny thing is that internet is significantly more common in the UK than in the US. Definitely one to backport; hooray for the demise of Internet I say. :-P